### DISCUSSION TITLE
BitWarden Web Vault Upgrade Timeline

### DISCUSSION THREAD

**Discussion by anthony-langford** [dani-garcia/vaultwarden#5064] [General]
Hello, first of all, thank you guys for the hard work put into making such a great lightweight BitWarden platform. I am wondering if there is a timeline for upgrading the [BitWarden Self-Host Web Vault](https://github.com/bitwarden/self-host) version. I noticed that they released an update to 2024.10.0 yesterday and tend to follow a monthly minor release cycle with 1-2 patch releases between.

I saw the current Vaultwarden web vault version is 2024.6.2c. Is there a target for number of versions to be lagging, or any possibility to get closer to the recent stable release? I am mostly wondering since these BitWarden releases can contain critical security fixes.

**Comment by BlackDex** [comment]
As mentioned in several other posts. We are stuck on 2024.6.2 currently, until the manager role migration functionality is going to be included into Vaultwarden.

There is a patch file for 2024.9.0 already, so that can be used to develop it. But the biggest issue is probably the whole migration of the manager role.

**Reply by anthony-langford** [reply]
Thank you for the prompt update and sorry for the repeat question. I did attempt a brief search for a similar post but didn't see any.

**Reply by BlackDex** [reply]
np, this title should be good for others to find hehe.
